lib/auth.js

```diff
@@ -6,12 +6,18 @@ var CookieJar = require('cookiejar');
 /**
  * SwaggerAuthorizations applys the correct authorization to an operation being executed
  */
-var SwaggerAuthorizations = module.exports.SwaggerAuthorizations = function () {
-  this.authz = {};
+var SwaggerAuthorizations = module.exports.SwaggerAuthorizations = function (authz) {
```

Comment:
I agree, I'm not much a fan of SwaggerAuthorization... but I do need something to do the following.. https://github.com/swagger-api/swagger-js/blob/master/lib/auth.js#L23-L74
And as a namespace, its fine. We won't be exposing it.